plugins/net_plugin/net_plugin.cpp

```diff
@@ -2324,7 +2333,9 @@ namespace eosio {
 
       c->last_handshake_recv = msg;
       c->_logger_variant.reset();
-      sync_master->recv_handshake(c,msg);
+      if (!c->is_gray) {
```

Comment:
I don't understand this change. For what and why?

Reply:
@afalaleev It is to do not sync if node is gray.